packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/methodSchema.js

```diff
@@ -0,0 +1,25 @@
+import SimpleSchema from "simpl-schema";
```

Comment:
Is there a reason these aren't defined in the top-level SimpleSchemas file?

Reply:
It was copied over from old plugin as such. I shall move this to `/src/simpleSchemas.js`